evap/student/templates/student_vote.html

```diff
@@ -56,11 +56,11 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
         {% csrf_token %}
 
         {% if small_evaluation_size_warning and not preview %}
-            <div class="card card-outline-warning mb-3">
-                <div class="card-header">
+            <div class="callout callout-warning mb-3">
+                <div class="callout-header">
                     <span class="fas fa-triangle-exclamation"></span> {% translate 'Small number of participants' %}
                 </div>
-                <div class="card-body tab-row">
+                <div class="callout-content tab-row">
                     {% blocktranslate %}Only a small number of people can take part in this evaluation. You should be aware that contributors might be able to guess who voted for a specific answer and who wrote a text answer. Results will only be published if two or more people participated in the evaluation.{% endblocktranslate %}
```

Comment:
the last sentence should be based on `VOTER_COUNT_NEEDED_FOR_PUBLISHING_RATING_RESULTS`